lib/statsd/instrument.rb

```diff
@@ -340,6 +340,11 @@ def backend
     @backend ||= StatsD::Instrument::Environment.default_backend
   end
 
+  def client
+    require 'statsd/instrument/client'
```

Comment:
Might want to but that require in a `begin/end`. Otherwise every access to `StatsD.client` will have to do a lookup in `$LOADED_FEATURES`, which isn't free.

Reply:
Fixed!